schema/schema.py

```diff
@@ -109,18 +108,65 @@ def generate_schema() -> dict:
     schema["type"] = "object"
     schema["properties"] = {
         # Version number required for samconfig files to be valid
-        "version": {"type": "number"}
+        "version": {"title": "Config version", "type": "number", "default": 0.1}
     }
     schema["required"] = ["version"]
     schema["additionalProperties"] = False
     # Iterate through packages for command and parameter information
     for package_name in _SAM_CLI_COMMAND_PACKAGES:
         new_command = retrieve_command_structure(package_name)
         commands.update(new_command)
-        for param_list in new_command.values():
+    for cmd_properties in commands.values():
+        for param_list in cmd_properties["parameters"].values():
```

Comment:
Lets use `.get()` instead so that we can specify an empty dictionary as the default just as an extra safe guard